samples/csharp/MicrosoftEduGraphSamples/workflows/Submission.cs

```diff
@@ -59,7 +58,7 @@ public void ReassignWorkflow()
                 graphClient = MicrosoftGraphSDK.GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["studentAccount"], _config["password"]);
 
                 // Get the student submission
-                var submissions = MicrosoftGraphSDK.Submission.GetSubmissions(graphClient, _config["classId"], assignmentId);
+                var submissions = MicrosoftGraphSDK.Submission.GetSubmissionsAsync(graphClient, _config["classId"], assignmentId);
```

Comment:
await across the board

Reply:
Thanks for your comment, currently we are using await in all our methods (is it required to put await in this case?) and when we try to put the await in workflows  I am getting error
![image](https://user-images.githubusercontent.com/114248043/200884176-31466b1c-ffff-4d13-85b6-759269f5b0c2.png)

Reply:
Thanks for your comment, currently we are using await in all our methods (is it required to put await in this case?) and when we try to put the await in workflows I am getting error
![image](https://user-images.githubusercontent.com/114248043/200884912-0486b9ce-cb4f-402a-8df3-d13abf3f3fdc.png)